plugins/modules/grafana_datasource.py

```diff
@@ -654,13 +668,23 @@ def create_datasource(self, data):
         url = "/api/datasources"
         self._send_request(url, data=data, headers=self.headers, method='POST')
 
+    def get_version(self):
```

Comment:
I had several bug report on modules where I was getting the version automatically because some user configure the Grafana server to make it not expose the version (security ...).

I don't know if it is wise to use the version sithout having a way to handle servers configured this way.

Maybe adding a `grafana_version` param for all module would allow us to cover both cases:
- by default we try to get the version automatically and behave accordingly
- if server is configured to not provide the info, users could provide the info in the module call.

WDYT ?

Reply:
Version check removed